gnovm/stdlibs/crypto/sha256/sha256.gno

```diff
@@ -1,12 +1,6 @@
 package sha256
 
-import (
-	isha256 "internal/crypto/sha256"
-)
-
 const Size = 32
 
 // Sum returns the SHA-256 checksum of the data.
-func Sum256(data []byte) [Size]byte {
-	return isha256.Sum256(data)
-}
+func Sum256(data []byte) [Size]byte
```

Comment:
```suggestion
func Sum256(data []byte) [Size]byte // injected
```

what about taking the habit to tell if something should be injected? it will make the reading of packages mixing injected and userland functions easier.

Reply:
or maybe make stdgen create a `packageName/inject_gen.gno`.